omero/util_scripts/Move_Annotations.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Moves Annotations from Images to their parent Wells."""
+
+import omero.scripts as scripts
+from omero.gateway import BlitzGateway
+from omero.model import WellAnnotationLinkI, WellI, ImageAnnotationLinkI
+from omero.rtypes import rstring, rlong
+
+
+ANN_TYPES = {
+    'Tag': 'TagAnnotationI',
+    'File': 'FileAnnotationI',
+    'Comment': 'CommentAnnotationI',
+    'Rating': 'LongAnnotationI',
+    'Key-Value': 'MapAnnotationI'
+}
+
+
+def move_well_annotations(conn, well, ann_type, remove_anns, ns):
+    """Move annotations from Images in this Well onto the Well itself."""
+    print "Processing Well:", well.id, well.getWellPos()
+    iids = [wellSample.getImage().id for wellSample in well.listChildren()]
+    print "  Image IDs:", iids
```

Comment:
Ahhhhh! The prints for scripts provide the log for the user. If you don't have any prints then they won't know what's gone wrong or won't have any "info" on what the script has done. They are essential.
At least where I've used a ```log()``` function to do the printing, it's only a case of adding back the print statement there, but for scripts with individual print statements this will be a fair bit harder.